src/config/server.rs

```diff
@@ -221,6 +226,8 @@ impl Server {
             cdn_user_agent: var("WEB_CDN_USER_AGENT")?
                 .unwrap_or_else(|| "Amazon CloudFront".into()),
             balance_capacity: BalanceCapacityConfig::from_environment()?,
+            cargo_compat_status_code_config: var_parsed("CARGO_COMPAT_STATUS_CODES")?
+                .unwrap_or(StatusCodeConfig::AdjustAll),
```

Comment:
if no value is set it will default to `AdjustAll` (i.e. convert everything to 200 OK)